src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx

```diff
@@ -39,7 +39,9 @@ const organizationNavigation = [
       {
         path: `${pathPrefix}/auth/`,
         title: t('Auth'),
-        show: ({access, features}) => features.has('sso') && access.has('org:admin'),
+        show: ({access, features}) =>
+          (features.has('sso') || features.has('sso-paywall-experiment')) &&
```

Comment:
It would be nice to show this to non-admins as well ("hey we have SSO too") without giving them the ability to actually activate it